integrations/electricity/energinet.js

```diff
@@ -114,15 +114,15 @@ async function connect({ requestToken }, logger) {
   }
 
   return {
-    refreshToken: token,
+    authToken: token,
     locationLon: lonLat[0],
     locationLat: lonLat[1],
   };
 }
 
 async function collect(state, logger) {
-  const { refreshToken, locationLon, locationLat } = state;
-  const accessToken = await getAccessToken(refreshToken);
+  const { authToken, locationLon, locationLat } = state;
+  const accessToken = await getAccessToken(authToken);
```

Comment:
Maybe change the argument name in `getAccessToken` as well `refreshToken` -> `authToken` as well before merging?